tests/typecheck/db/models/test_fields_choices.yml

```diff
@@ -11,6 +11,10 @@
     main:4: note:         def __iter__(self) -> Iterator[Union[Tuple[Any, Any], Tuple[str, Iterable[Tuple[Any, Any]]]]]
     main:4: note:     Got:
     main:4: note:         def __iter__(self) -> Iterator[str]
+    main:4: note:     Expected:
+    main:4: note:         def __iter__(self) -> Iterator[Union[Tuple[Any, Any], Tuple[str, Iterable[Tuple[Any, Any]]]]]
+    main:4: note:     Got:
+    main:4: note:         def __iter__(self) -> Iterator[str]
```

Comment:
This is annoying and perhaps a new bug in mypy v1.16? It doubles this note for some reason.

Reply:
Yes, looks like so. Please, report upsteam.